pulseapi/profiles/serializers.py

```diff
@@ -162,6 +167,7 @@ def to_representation(self, obj):
         return {
             'id': obj.id,
             'name': obj.name
```

Comment:
always end every pair in a dict with a comma

Reply:
done